.github/workflows/flake8.yml

```
@@ -20,7 +20,7 @@ jobs:
           git remote add upstream https://github.com/pvlib/pvlib-python.git
           git fetch upstream $GITHUB_BASE_REF
       - name: Run Flake8 linter
-        run: git diff upstream/$GITHUB_BASE_REF HEAD | flake8
+        run: git diff upstream/main -- "*.py" | flake8
```

```suggestion
        run: git diff upstream/$GITHUB_BASE_REF HEAD -- "*.py" | flake8
```

We should probably keep this part, just in case we ever start sending PRs to branches other than `main`.  Sorry I didn't think to do that in the original suggestion!